xdsl/rewriter.py

```diff
@@ -262,16 +215,6 @@ def insert_ops_at_location(ops: Sequence[Operation], insertion_point: InsertPoin
         else:
             insertion_point.block.add_ops(ops)
 
-    @staticmethod
-    def insert_op_after(op: Operation, new_op: Operation):
-        """Inserts a new operation after another operation."""
-        Rewriter.insert_ops_at_location((new_op,), InsertPoint.after(op))
-
-    @staticmethod
-    def insert_op_before(op: Operation, new_op: Operation):
```

Comment:
Please note that this also removes a weird feature of our APIs, where `Rewriter` and `PatternRewriter` flip the order of the `target` op and `new_op` in `inser_op_before`, the `InsertPoint` removes the possibility of ambiguity about which is which.
#859

Reply:
I Just stumbled on this last week again. Thank you so much :heart: